flow360/component/flow360_params/solvers.py

```diff
@@ -185,74 +266,78 @@ class TurbulenceModelSolver(GenericFlowSolverSettings):
 
     Example
     -------
-    >>> ts = TurbulenceModelSolver(model_type='SA', absolute_tolerance=1e-10)
+    >>> ts = TurbulenceModelSolver(absolute_tolerance=1e-10)
     """
 
-    model_type: TurbulenceModelModelType = pd.Field(
-        alias="modelType", default=TurbulenceModelModelType.SA
+    model_type: str
```

Comment:
is it abstract class? If not, should the model_type by Literal here?

Reply:
Made the class abstract (with the metaclass argument) since its never instantiated